ebuild-writing/misc-files/metadata/text.xml

```diff
@@ -68,6 +68,7 @@ metadata.xml:
     <c>&lt;longdescription&gt;</c>,
     <c>&lt;maintainer&gt;</c>,
     <c>&lt;slots&gt;</c>,
+    <c>&lt;stabilize-allarches&gt;</c>,
```

Comment:
I wonder if we should make it `<.../>` since it's always empty.

Reply:
I'd rather not, since we're also omitting the closing tag for all other elements. (I guess it would be more systematic to list the element names without the brackets, but that doesn't make it any more readable either.)

Reply:
You could also use `<.../>` for everything.

Reply:
I never understood why one would write `<.../>`, which is a shortcut for `<...></...>` and denotes an empty element, even if that element (e.g., `maintainer`) cannot be empty. Then again, I'm not an XML person.

If we go for strictness then we should say "element" in the table header and omit the brackets. But I believe readability is better if we leave them in place, even if that doesn't strictly distinguish the element from its opening tag.